clisops/core/average.py

```diff
@@ -78,7 +78,7 @@ def average_shape(
     try:
         from xesmf import SpatialAverager
     except ImportError:
-        raise ValueError("Package xesmf >= 0.5.0 is required to use average_shape")
+        raise ValueError("Package xesmf >= 0.6.2 is required to use average_shape")
```

Comment:
```suggestion
        raise ValueError("Package xesmf is required to use average_shape")
```

Reply:
The required version is set in the pyproject.toml file. 

I've tried to run the test suite with an older xesmf version and had all kinds of problems with dependencies.